ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "task"
-version = "2.9.0"
```

Comment:
As this is a new feature, shouldn't this be a minor release ?

Reply:
Yes, I'll update it